docs/reference/modules/indices/circuit_breaker.asciidoc

```diff
@@ -115,3 +115,13 @@ documentation for more information.
     Limit for the number of unique dynamic scripts within a certain interval
     that are allowed to be compiled. Defaults to 75/5m, meaning 75 every 5 
     minutes.
+
+==== Operating Limits of Elasticsearch Clusters
+
+Every Elasticsearch cluster has a limit to the amount of data it can store. When this limit is exceeded, the cluster will become inoperable and every attempted data operation will fail with some kind of circuit-breaker exception. This is not just a question of having so much data stored on disk that every query becomes unacceptably slow: when the operational limit is exceeded, the cluster will be unable to perform any kind of query, ingestion or indexing operation. Every attempted user operation - and also many administrative operations will fail with circuit-breaker exceptions.
```

Comment:
Please wrap lines to 140 characters are outlined in https://github.com/elastic/elasticsearch/blame/master/CONTRIBUTING.md#L187?
```suggestion
Every {es} cluster has a limit to the amount of data it can store. When this limit is exceeded, the cluster becomes inoperable and every attempted data operation will fail with some kind of circuit breaker exception. This is not just a question of having so much data stored on disk that every query becomes unacceptably slow; when the operational limit is exceeded, the cluster will be unable to perform any kind of query, ingestion or indexing operation. Every attempted user operation  and many administrative operations will fail with circuit breaker exceptions.
```